src/olympia/amo/search.py

```diff
@@ -14,7 +14,7 @@
 DEFAULT_TIMEOUT = 5
 
 
-def get_es(hosts=None, timeout=None, **settings):
```

Comment:
`**settings` was never passed anywhere that I could see

Reply:
we don't ever appear to pass `hosts` either, fwiw - and the one place we pass `timeout` as an arg it's `settings.ES_TIMEOUT` which is the default anyway

Reply:
And now I'm looking, the `getattr(...` fallbacks with the constants in this file are also redundant - we define `ES_HOSTS` and `ES_TIMEOUT` in settings_base so those settings are also present.